src/layouts/supervisor-error-screen.ts

```diff
@@ -43,7 +43,7 @@ class SupervisorErrorScreen extends LitElement {
               <li>
                 <a
                   class="supervisor_error-link"
-                  href="http://homeassistant.local:4357"
+                  .href=${`http://${location.hostname}:4357`}
```

Comment:
This will (also) not work in a lot of cases, for example if you visit your instance through a nabucasa remote url.

Reply:
The only way for us to know the local hostname, is asking the supervisor API, but if we are on this page, that will likely not work.